File: react/src/api/models/acroStack/openIddictManagement/dtos/OpenIddictAuthorizationDto.ts
/* oxlint-disable */

/**
 * @description Authorization view for admin listing. Deliberately excludes the\r\nProperties column (consent/principal JSON) to keep the surface minimal.
 * @type object
 */
export type AcroStackOpenIddictManagementDtosOpenIddictAuthorizationDto = {
  /**
   * @description
   * Format: `uuid`
   * @type string | undefined
   */
  id?: string;
  /**
   * @description
   * Format: `uuid`
   * @type string | undefined
   */
  applicationId?: string | null;
  subject?: string | null;
  type?: string | null;
  status?: string | null;
  /**
   * @description
   * Format: `date-time`
   * @type string | undefined
   */
  creationDate?: string | null;
};

File: modules/AcroStack.OpenIddictManagement/Dtos/OpenIddictAuthorizationDto.cs
using System;

namespace AcroStack.OpenIddictManagement.Dtos;

/// <summary>
/// Authorization view for admin listing. Deliberately excludes the
/// Properties column (consent/principal JSON) to keep the surface minimal.
/// </summary>
public class OpenIddictAuthorizationDto
{
    public Guid Id { get; set; }

    public Guid? ApplicationId { get; set; }

    public string Subject { get; set; } = default!;

    public string Type { get; set; } = default!;

    public string Status { get; set; } = default!;

    public DateTime? CreationDate { get; set; }
}
